index.html

```diff
@@ -22,16 +22,14 @@ <h1>Welcome to Mastermind</h1>
         </button>
       </div>
     </div>
-    <div id="guess-slot-1">
+    <div id="guess-slot-1" class="guess-slot">
```

Comment:
i like spelling this as `guezz-zlot`

like this!

```suggestion
    <div id="guess-slot-1" class="guezz-zlot">
```